src/components/Settings/SettingsAccount.js

```diff
@@ -0,0 +1,100 @@
+import {Text, View} from "react-native";
+import {viewStyles, textStyles} from "../../styles/settings/settings";
+import React from "react";
+import {Picker} from "@react-native-picker/picker";
+import {colors} from "../../styles/global";
+import Pressable from "react-native/Libraries/Components/Pressable/Pressable";
+import CheckBox from "@react-native-community/checkbox";
+import {inatLanguages} from "../../dictionaries/languages";
```

Comment:
Not that I'm aware of - @pleary?

Reply:
There are a few reasons we hard-code these in the Android app, but the most important is that we don't want the user choosing a language the app does not support and looking for a file that doesn't exist. I'm pretty sure we can guard against that in this app b/c it is only loading files that do exist, and if another locale is requested we'll just load English, but I agree it would be far preferable to show exactly the same choices we show on the web. So here's a plan

* @budowski: remove time zones from this PR as we are no longer supporting user-editable time zones
* I'll will make issues to have the API serve up locales and language names the app is allowed to use, but for now we can just use the approach in this PR
* For Sites / the Network, we should not hardcode them and use https://api.inaturalist.org/v2/sites instead, BUT since that was never specified here, let's just ignore that here and work on it in the future; I'll make an issue.